### PR TITLE
Add specific conditions in Geist

### LINKED ISSUE #277: Fix: Geist Lender Borrower
As the pattern is "GeistStableLenderStable", and we have other strategies where name includes "Geist", we will need to add specific conditions in this case.

Please modify the current criteria [here](https://github.com/yearn/yearn-watch/blob/master/src/utils/config/risk-framework.json#L830) and add `GeistMIMLender`, `GeistDAILender`, `GeistfUSDTLender` and `GeistfUSDCLender`.

Please verify that the strategies are not included in the Other groups. For example: [this is](https://github.com/yearn/yearn-watch/blob/master/src/utils/config/risk-framework.json#L857) the old value.

### PR DESCRIPTION
**Pull request by magikadev**
#277

### CODE REVIEW

File: src/utils/config/risk-framework.json [hunk 1]
```diff
@@ -827,7 +827,7 @@
             "complexityScore": 4,
             "teamKnowledgeScore": 2,
             "criteria": {
-                "nameLike": ["GeistLenderBorrower"],
+                "nameLike": ["GeistLenderBorrower", "GeistMIMLender", "GeistDAILender", "GeistfUSDTLender", "GeistfUSDCLender"],
```
Comment (rareweasel) [inline src/utils/config/risk-framework.json]: We can remove `GeistLenderBorrower` from the list.

File: src/utils/config/risk-framework.json [hunk 2]
```diff
@@ -854,7 +854,11 @@
                     "veLp_arb_Solidex",
                     "fbeets compounder",
                     "genlevgeist",
-                    "GeistLenderBorrower"
+                    "GeistLenderBorrower",
```
Comment (rareweasel) [inline src/utils/config/risk-framework.json]: Also here.

**Review by rareweasel — CHANGES_REQUESTED** [review]

File: src/utils/config/risk-framework.json (hunk 2 above)
Comment (magikadev) [reply src/utils/config/risk-framework.json]: Resolved in https://github.com/yearn/yearn-watch/pull/281/commits/ed4fdc112d8b4aad30cc53acaf3687df1b95334b

File: src/utils/config/risk-framework.json (hunk 1 above)
Comment (magikadev) [reply src/utils/config/risk-framework.json]: Resolved in https://github.com/yearn/yearn-watch/pull/281/commits/ed4fdc112d8b4aad30cc53acaf3687df1b95334b

**Pull request merged**